user-guide/Basic_Functionality/Trending/Advanced_analytics_trending.md

```diff
@@ -16,6 +14,7 @@ These features are:
 
 - [Working with pattern matching](xref:Working_with_pattern_matching)
 
-## Cloud-Based Advanced Analytics
```

Comment:
You've removed this again.
Does it make sense to have 'Adding related parameters to a trend graph' in the same list?
Because it isn't provided by SLAnalytics, but by the ModelHost DxM.
This cannot be deactivated via *System Center* > *System settings* > *analytics config* as well.